test/games/strategy/engine/message/EndPointTest.java

```diff
@@ -6,10 +6,9 @@
 import junit.framework.TestCase;
 
 public class EndPointTest extends TestCase {
-  @SuppressWarnings("rawtypes")
   public void testEndPoint() {
     final EndPoint endPoint = new EndPoint("", Comparator.class, false);
-    endPoint.addImplementor(new Comparator() {
+    endPoint.addImplementor(new Comparator<Object>() {
```

Comment:
This one you should be able to replace with a lamdba:
endPoint.addImplementor((o1,o2) -> 2);

The types of o1 and o2 are inferred, but should be 'EndPoint'. Meaning the code should compile with  `new Comparator<Endpoint>`